rules/linux/defense_evasion_busybox_binary.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2022/03/15"
+maturity = "production"
+updated_date = "2022/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary busybox abuse to break out from restricted environments by spawning an interactive system
+shell.This activity is not standard use with this binary for a user or system administrator. It indicates a potentially
+malicious actor attempting to improve the capabilities or stability of their access.
```

Comment:
Maybe we can elaborate a less generic description, describe what the binary does, etc

Reply:
Response in https://github.com/elastic/detection-rules/pull/1840